adapter/src/test/java/org/apache/calcite/rex/InListConversionTest.java

```diff
@@ -0,0 +1,92 @@
+/* Copyright 2020 Twilio, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.rex;
+
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class InListConversionTest extends RexProgramTestBase {
+
+  @Before
+  public void setUp() {
+    super.setUp();
+  }
+
+  @Test
+  public void oneFieldOredTogether() {
+    final RexNode aRef = input(tInt(), 0);
+    // $0 = 0 OR $0 = 1 OR $0 = 2
+    RexNode expr = or(eq(aRef, literal(0)), eq(aRef, literal(1)), eq(aRef, literal(2)));
+    final String simplified = "SEARCH($0, Sarg[0, 1, 2])";
+    final String expanded = "OR(=($0, 0), =($0, 1), =($0, 2))";
+    checkSimplify(expr, simplified).expandedSearch(expanded);
+  }
+
```

Comment:
The tests below currently fail, I don't think they matter since we will represent an IN list as an SArg directly instead of converting to an OR list as we did previously.

Reply:
Sorry I don't understand. Can you expand further on this?

Reply:
I have removed this test, we can use  [RelBuilder.in](https://github.com/apache/calcite/blob/67ece68208af8385c9e06b6b5cc812d6ad36293c/core/src/main/java/org/apache/calcite/tools/RelBuilder.java#L674) to directly represent an IN clause which will be converted to an SArg (instead of converting the IN clause to an OR clause).

Reply:
K so these tests were attempting to make a disjunctions and make them into InList predicates. We are kindof meh because it is complex.